classroom.rb

```diff
@@ -0,0 +1,60 @@
+# Represents a classroom with attributes such as label and students
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a student with attributes name and classroom.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
```

Comment:
- [ ] Please create a method in the `Student` class that handles the _**belongs-to**_ part of the association between the `Classroom` and `Student` classes. This will ensure that `Student` class instances can have the ability to record the `Classroom` they belong to from their end. 👍 💻 

#### Hint: In the `Student` class that you created in the `student.rb` file, you can create a method like this:

```ruby
def classroom=(classroom)
  @classroom = classroom
  classroom.students.push(self) unless classroom.students.include?(self)
end
```

Reply:
@Gambit142 done